java/ql/src/semmle/code/java/dataflow/internal/DataFlowPrivate.qll

```diff
@@ -137,10 +137,10 @@ class MapValueContent extends Content, TMapValueContent {
  * Thus, `node2` references an object with a field `f` that contains the
  * value of `node1`.
  */
-predicate storeStep(Node node1, Content f, PostUpdateNode node2) {
+predicate storeStep(Node node1, Content f, Node node2) {
```

Comment:
Just verifying that this change is expected.

Reply:
Indeed, because we now have storeSteps pointed at synthetic nodes that don't have PUNs (perhaps they should?)